src/plugins/kibana_usage_collection/server/collectors/management/telemetry_management_collector.ts

```diff
@@ -39,7 +38,410 @@ export function registerManagementUsageCollector(
     type: 'stack_management',
     isReady: () => typeof getUiSettingsClient() !== 'undefined',
     fetch: createCollectorFetch(getUiSettingsClient),
-    schema: stackManagementSchema,
+    schema: {
+      // sensitive
+      'timelion:quandl.key': {
```

Comment:
I had to move the schema to the main collector definition to support the `_meta` property.